server/src/main/resources/application.properties

```diff
@@ -73,6 +73,9 @@ conductor.default-event-queue.type=sqs
 conductor.app.workflow-execution-lock-enabled=false
 conductor.workflow-execution-lock.type=noop_lock
 
+#Used to enable/disable the duplicate message delivery persistence
+conductor.app.eventExecutionPersistenceEnabled=true
```

Comment:
Should we keep default value to false?

Reply:
+1 to preserve backwards compatibility this should be false.